components/AddLink.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function AddLink() {
+  return (
+    <>
+      <StyledLink href="/add">
+        <p>Didnt find your Plant? Create your own!</p>
+      </StyledLink>
+    </>
+  );
+}
```

Comment:
I would recommend to place this code directly where you need it, it does not really hide enough complexity to justify the extra component. You can still keep your styled component here and name it AddLink if you like. You can export import the styled components directly.